Makefile

```diff
@@ -12,10 +12,12 @@ ifeq ($(OS), darwin)
 OPENSSL_DIR ?= $(shell brew --prefix)/opt/openssl
 GRPCURL_OS ?= osx
 YAML_DIR ?= $(shell brew --prefix)/opt/libyaml
+CURL_INCDIR ?= /usr/include/x86_64-linux-gnu/
 else
 OPENSSL_DIR ?= /usr
 GRPCURL_OS ?= $(OS)
 YAML_DIR ?= /usr
+CURL_INCDIR ?= /usr/include/x86_64-linux-gnu/
```

Comment:
I also feel a bit uneasy about this. It seems it puts architecture here. Might work for now, but might cause issues later.

Reply:
I agree, but didn't immediately know a better solution...

Reply:
Ping @fffonion do you know better way here? Or should we go with this. I'd like to get this PR in finally.

Reply:
I had to do something similar in the upgrade tests, so it seems this is the way for now.